generators/ruby/sdk/CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## [0.0.2] - 2024-02-01
+
+- Improvement: Support output mode block configuration
```

Comment:
can we add a snippet of the schema here? so that you can directly link folks to the example. 

Added `rubygems` output block to generators.yml. To publish your ruby gem, setup your generators.ym like this: 
```
# generators.yml 

groups: 
 - name: ...
    version: ...
    location: 
       output: rubygems
       api-key: ${API_KEY}
       gem-name: "petstore"
```